Tests/RichTextKitTests/Data/RichTextDataFormatTests.swift

```diff
@@ -12,45 +12,43 @@ import XCTest
 
 final class RichTextDataFormatTests: XCTestCase {
 
-    let vendorType = UTType(exportedAs: "fooType")
+    private let vendorType = UTType(exportedAs: "fooType")
 
-    lazy var vendorFormat = RichTextDataFormat.vendorArchivedData(
+    private lazy var vendorFormat = RichTextDataFormat.vendorArchivedData(
         id: "foo",
         fileExtension: "fooFile",
         fileFormatText: "Foo File (*.foo)",
         uniformType: vendorType
     )
 
-    func idResult(for format: RichTextDataFormat) -> String {
+    private func idResult(for format: RichTextDataFormat) -> String {
```

Comment:
I'd love to move these to a private extension to avoid having to type private everywhere :D

Reply:
Let's do that!

Reply:
✅ done in https://github.com/danielsaidi/RichTextKit/pull/96/commits/7c6ebb322d6d5075796df93153cba552b8f69165

Reply:
Amazing, thank you!